tests/legacy-cli/e2e/utils/process.ts

```diff
@@ -96,39 +96,47 @@ function _exec(options: ExecOptions, cmd: string, args: string[]): Promise<Proce
   _processes.push(childProcess);
 
   // Create the error here so the stack shows who called this function.
+  let error: Error;
```

Comment:
Wouldn't the below do the same?

```suggestion
  const error = new Error();
```

Reply:
Looks like you're correct. I always thought the stack was where the error was thrown, but it's actually just where it was constructed?

Reply:
That is correct. Can also change the `catch` to which is more readable. 

```ts
  }).catch((err) => {
    // Create the error here so the stack shows who called this function.
    return Promise.reject(new Error(err));
  });
```

Reply:
If you do the `new Error` in the catch callback doesn't that mean the stack trace will be just the callback?

Reply:
right